### PR TITLE
Plant phenomics assembly

### PR DESCRIPTION
**Pull request by erlefloch**
Hi,
This is a first version of a plant phenomics assembly page, made thanks to the contributors presenting the different tools. 
I would like to talk about it at IPPS next week.
Feel free to edit or comment,
Cheers
Erwan

### CODE REVIEW

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+</details>
+
+##### PIPPA 
+
+[PIPPA](https://pippa.psb.ugent.be/), the PSB Interface for Plant Phenotype Analysis, is the central web interface and database that provides the tools for the management of the plant imaging robots on the one hand, and the analysis of images and data on the other hand. The database supports all MIAPPE fields which are accessible through the BrAPI endpoints. Experiment pages are marked up with Bioschemas to improve findability on google. 
+
+### Data processing and analysis 
+
+It is important to keep in mind the difference between data processing and analysing.
+[Processing](https://rdmkit.elixir-europe.org/processing) provides the tools and procedures to transform primary data, such as imaging or observational data, to appropriate quality and processability:
```
Comment (bedroesb) [inline pages/tool_assembly/plant_phenomics_assembly.md:90]:
```suggestion
[Processing](processing) provides the tools and procedures to transform primary data, such as imaging or observational data, to appropriate quality and processability:  
```

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+##### PIPPA 
+
+[PIPPA](https://pippa.psb.ugent.be/), the PSB Interface for Plant Phenotype Analysis, is the central web interface and database that provides the tools for the management of the plant imaging robots on the one hand, and the analysis of images and data on the other hand. The database supports all MIAPPE fields which are accessible through the BrAPI endpoints. Experiment pages are marked up with Bioschemas to improve findability on google. 
+
+### Data processing and analysis 
+
+It is important to keep in mind the difference between data processing and analysing.
+[Processing](https://rdmkit.elixir-europe.org/processing) provides the tools and procedures to transform primary data, such as imaging or observational data, to appropriate quality and processability:  
+[Analysis](https://rdmkit.elixir-europe.org/analysing), on the other hand, is concerned with extracting information from the processed data for the purpose of supporting knowledge acquisition:
```
Comment (bedroesb) [inline pages/tool_assembly/plant_phenomics_assembly.md:91]:
```suggestion
[Analysis](analysing), on the other hand, is concerned with extracting information from the processed data for the purpose of supporting knowledge acquisition: 
```

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+### Machine actionable data sharing
+
+The Breeding API ([BrAPI](http://www.brapi.org)) is a MIAPPE compliant web service specification available on several [deposition databases](https://www.brapi.org/servers). Those endpoints can be validated using the BrAPI validator  [BRAVA](https://webapps.ipk-gatersleben.de/brapivalidator/). BrAPI hosts several documentation and training material to support its usage.
+
+[FAIDARE](https://bio.tools/faidare) is a portal allowing users to search data across distinct databases.
+
+### Data preservation
+
+No central, global repository with long term [data preservation](https://rdmkit.elixir-europe.org/preserving) exists for Plant Phenotyping data. This is under the responsibility of each country.
```
Comment (bedroesb) [inline pages/tool_assembly/plant_phenomics_assembly.md:119]:
```suggestion
No central, global repository with long term [data preservation](preserving) exists for Plant Phenotyping data. This is under the responsibility of each country.
```

**Review by bedroesb — COMMENTED** [review]
Good first attempt! 

General comment: internal RDMkit pages are linked to by just using the page name as you will see in my suggestions. Can you also look into tagging the tools you mention in the page with the page_id you propose [here](https://docs.google.com/spreadsheets/d/16RESor_qQ_ygI0lQYHR23kbZJUobOWZUbOwhJbLptDE/edit#gid=268211668)

**Comment by bedroesb** [comment]
PIPPA btw is part of the Belgian page and should get tagged in the front matter of the national markdown, I don't know for PHIS ;)

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
+---
+title: Plant Phenomics
+contributors: [Anne-Françoise Adam-Blondon, Cyril Pommier, Bert Droesbeke, Matthias Lange, Daniel Arend, Daniel Faria, Isabelle Alic, Philippe Rocca-Serra, Sebastian Beier, Erwan Le Floch, did I miss someone ?]
+description: Tool assembly for managing plant phenomic data.
+page_id: plant pheno assembly
+affiliations:
+related_pages: 
+  your_tasks: [metadata, data publication]
+  your_domain: [plants]
+faircookbook:
```
[the remark this answers is not in the document]
Comment (erlefloch) [reply pages/tool_assembly/plant_phenomics_assembly.md:10]: alright

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+* [MIAPPE template](https://github.com/MIAPPE/MIAPPE/tree/master/MIAPPE_Checklist-Data-Model-v1.1/MIAPPE_templates)
+* [MIAPPE ISA template (under validation)](https://github.com/MIAPPE/MIAPPE/blob/eppn2020-template/MIAPPE_Checklist-Data-Model-v1.1/MIAPPE_templates/MIAPPEv1.1_ISA_EPPN2020_template.xlsx )
+</details>
+
+
+### Experimental data gathering and management 
+
+#### File based systems
+
+##### FAIRDOM-Seek
```
[the remark this answers is not in the document]
Comment (erlefloch) [reply pages/tool_assembly/plant_phenomics_assembly.md:55]: I will try with bold links

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+affiliations:
+related_pages: 
+  your_tasks: [metadata, data publication]
+  your_domain: [plants]
+faircookbook:
+---
+
+## What is the plant phenomics tool assembly and who can use it ?
+
+The plant phenomics tool assembly covers the whole [life cycle](https://rdmkit.elixir-europe.org/data_life_cycle) of experimental plant phenotyping data.  It uses the concepts of the [MIAPPE](https://www.miappe.org/) (Minimum Information About a Plant Phenotyping Experiment) standard. A general introduction of MIAPPE concepts can be found in the following [webinar](https://tess.elixir-europe.org/materials/plant-phenotyping-data-managment-webinar-miappe).
```
[the remark this answers is not in the document]
Comment (erlefloch) [reply pages/tool_assembly/plant_phenomics_assembly.md]: that's right, do you mean to cross reference to the #more-information section?

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+The plant phenomics tool assembly helps [researchers](https://rdmkit.elixir-europe.org/researcher), [research data stewards](https://rdmkit.elixir-europe.org/data_steward_research) and [developers in charge of infrastructures](https://rdmkit.elixir-europe.org/data_steward_infrastructure) supporting plant phenotyping data management  to enable:
```
[the remark this answers is not in the document]
Comment (erlefloch) [reply pages/tool_assembly/plant_phenomics_assembly.md]: would you remove the links, or the sentence ?

File: pages/tool_assembly/plant_phenomics_assembly.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (erlefloch) [reply pages/tool_assembly/plant_phenomics_assembly.md]: I removed it, it was indeed redundant

File: pages/tool_assembly/plant_phenomics_assembly.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (erlefloch) [reply pages/tool_assembly/plant_phenomics_assembly.md]: I simplified the sentence as all the roles were present indeed.

**Comment by erlefloch** [comment]
> PIPPA btw is part of the Belgian page and should get tagged in the front matter of the national markdown, I don't know for PHIS ;)

do you mean, PHIS should be added to the tools and resources section on the French page ?

**Comment by erlefloch** [comment]
Thanks for your feedback @bedroesb and @korbinib !
I modified the text accordingly. I also updated the tool_and_resource_list sheet, and I added the info to link to MIAPPE training material in TeSS.

**Comment by bedroesb** [comment]
> > PIPPA btw is part of the Belgian page and should get tagged in the front matter of the national markdown, I don't know for PHIS ;)
> 
> do you mean, PHIS should be added to the tools and resources section on the French page ?

If not already done, yes! :)

**Comment by erlefloch** [comment]
hi everyone, sorry I messed up the review requests by clicking everywhere, sorry and don't pay attention :)

**Comment by bedroesb** [comment]
@erlefloch is PHIS not build to serve the French institutes like PIPPA, or am I wrong? Otherwise I would move it out of the main tools table to the country page metadata instead of linking towards it.

**Comment by bedroesb** [comment]
I also took PIPA out of the main tools table since it is already described on our national page

**Comment by erlefloch** [comment]
> @erlefloch is PHIS not build to serve the French institutes like PIPPA, or am I wrong? Otherwise I would move it out of the main tools table to the country page metadata instead of linking towards it.

From what I understood, PHIS is the name of both the software and of an instance of it. The software itself can be deployed anywhere, for example I've seen recently [here](https://www.npec.nl/phenotyping-modules/module-7-data/) that it was in use in Netherlands

**Comment by bedroesb** [comment]
@erlefloch ok fair enough! For me it looks good now.

**Review by bedroesb — APPROVED** [review]

**Comment by bedroesb** [comment]
@korbinib This page would be presented this Monday and it would be nice if a first version could be merged. I don't know If you think your comments got implemented and this disserves your approval ? :) Later on you might want to apply your tricks to include the font in the SVG

**Comment by bedroesb** [comment]
@erlefloch I just clicked on the GitHub icon on the PHIS website, and it goes to [Open SILEX](https://github.com/OpenSILEX/). Maybe PHIS is a deployment of  OpenSIlex?

**Review by korbinib — APPROVED** [review]

**Comment by korbinib** [comment]
@bedroesb @erlefloch  Thank you!

**Comment by erlefloch** [comment]
> @erlefloch I just clicked on the GitHub icon on the PHIS website, and it goes to [Open SILEX](https://github.com/OpenSILEX/). Maybe PHIS is a deployment of OpenSIlex? That one for sure seems to disserve a spot in the main tools table.

Yes probably ! @cpommier what was the relationship between OpenSILEX and PHIS again ?

**Comment by bedroesb** [comment]
I am intrested in the answer for sure ;) Will upload the tools to RDMkit as they are now

**Comment by cpommier** [comment]
HI, sorry for the delayed answer and thanks a lot for your  high activity :)
Opensilex is the basis software, that can be implemented in several flavors: PHIS, SIXTINE, VitisOno, etc...  
Then each of those flavors are deployed on different sites. 
PHIS is a flavor dedicated to high throughput phenotyping, like PIPPA, but is getting used beyond France. 
If I understand the logic well, we should put forward PHIS in the general tools section as it goes beyond France. Linking this to phis website, that links to opensilex github, seems ok to me. @Isabelle-inrae could you confirm that this is fine with you ?

**Comment by cpommier** [comment]
This has ben merged ? It is therefore live now ?

**Comment by bedroesb** [comment]
@cpommier when #994 is merged it will be live in the way you described! I just wonder if it wis worthed to mention Opensilex somewhere on RDMkit. I never heard of it.

**Comment by Isabelle-inrae** [comment]
Hi @bedroesb, I confirm @cpommier answer; I think it's better to put PHIS forward! Thanks

**Comment by bedroesb** [comment]
Thanks for responding!

**Pull request merged**